docs/src/content/docs/guides/delivering-remote-images.mdx

```diff
@@ -0,0 +1,272 @@
+---
+title: Delivering Remote Images
+description: Fetching and optimizing remote images with Astro Cloudinary
+head:
+  - tag: title
+    content: Delivering Remote Images - Astro Cloudinary
+---
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+import { CldImage } from '../../../../../astro-cloudinary';
+
+import CodeBlock from '../../../components/CodeBlock.astro';
+import HeaderImage from '../../../components/HeaderImage.astro';
+
+
+Astro Cloudinary provides two powerful methods for fetching, optimizing, and delivering remote images through Cloudinary's transformation pipeline: the Fetch API and Auto Upload Mapping. These features are particularly useful when you want to fetch and optimize external images without explicitly storing them in your Cloudinary account.
+
+## Method 1: Using the Fetch API
+
+The Fetch API allows you to deliver remote images on-the-fly through Cloudinary, applying transformations as needed. This method is ideal for dynamically accessing and transforming images from external sources.
+
+### Basic Usage
+
+To fetch a remote image, you need to include the full URL of the image in the `src` prop and set `deliveryType="fetch"`.
+
+<HeaderImage>
+<CldImage
+src="https%3A%2F%2Fupload.wikimedia.org%2Fwikipedia%2Fcommons%2F4%2F47%2FPNG_transparency_demonstration_1.png"
```

Comment:
@RaghavMangla the URL already gets encoded, so by encoding it here, it doesn't work

Can you update the examples to use a non-decoded URL?


<img width="822" alt="image" src="https://github.com/user-attachments/assets/c35553e9-0e7d-4f59-b5e1-3d5ad8acab63">